stdlib/public/core/StringStorage.swift

```diff
@@ -114,6 +139,15 @@ final internal class _StringStorage: _AbstractStringStorage {
     @inline(__always) set { _countAndFlags.count = newValue }
   }
 
+  // The total allocated storage capacity. Note that this includes the required
+  // nul-terminator
+  @nonobjc
+  fileprivate var _realCapacity: Int {
```

Comment:
Not private?

Reply:
Ah, it looks like I still live in the dystopian world of SE-0025. However, I don't think private is right here; internal fits the context better. I'll change it to that.